packages/react/src/control-base/ControlBase.tsx

```diff
@@ -29,7 +29,6 @@ export const ControlBase = forwardRef<HTMLDivElement, ControlBaseProps>(
   ) => {
     const { restProps, sprinkleProps } = extractSprinkles(props);
     const id = useId(idProp);
-
```

Comment:
Please keep the bank line here

We should always use blank lines to group related blocks of code - kinda like paragraphs in text

Reply:
My fault vai, I added a log in this line, later removed the log and it happened.

Reply:
can you bring back the blank line in that case please

Reply:
also needs a rebase with `main`